qubesadmin/tools/qvm_shutdown.py

```diff
@@ -54,6 +54,8 @@
 def main(args=None, app=None):  # pylint: disable=missing-docstring
     args = parser.parse_args(args, app=app)
 
+    force = bool(args.all_domains)
```

Comment:
Having this option supported in the backend, it would be useful to also allow using it directly via --force argument, not only implied by --all.